app/move/validators.js

```diff
@@ -33,4 +33,13 @@ module.exports = {
       Controller.validators.regex(value, /^[A-Z][0-9]{4}[A-Z]{2}$/i)
     )
   },
+  policeNationalComputerNumber(value) {
+    return (
+      value === '' ||
+      Controller.validators.regex(
```

Comment:
Do we need to check for the checksum too? https://dsdmoj.atlassian.net/wiki/spaces/PECS4/pages/3082551297/PNC+Validation

Reply:
Yes please!